tests/migrations/test_commands.py

```diff
@@ -595,6 +597,29 @@ def test_makemigrations_empty_connections(self):
                 init_file = os.path.join(migration_dir, '__init__.py')
                 self.assertTrue(os.path.exists(init_file))
 
+    def test_makemigrations_other_datbase_is_readonly(self):
+        """
+        Ensure makemigrations doesn't choke on the non-default database being read-only. #27054
+        """
+
+        def patched_ensure_schema(migration_recorder):
+            from django.db import connections
+            if migration_recorder.connection is connections['other']:
+                raise MigrationSchemaMissing()
+            else:
+                return mock.DEFAULT
+
+        self.assertTableNotExists("migrations_unicodemodel")
+        apps.register_model('migrations', UnicodeModel)
```

Comment:
It's an okay test, I suppose, but I'd lean toward making it in a separate commit since it's testing more than just the original issue. What do you think?